src/common/index.ts

```diff
@@ -0,0 +1 @@
+export * from './abstract/abstractActiveSeats';
```

Comment:
why does this "common" directory exists?

Reply:
Done